plugins/fetchinvoice.c

```diff
@@ -1105,6 +1175,34 @@ static struct command_result *json_sendinvoice(struct command *cmd,
 					    "quantity parameter unnecessary");
 	}
 
+	/* BOLT-offers #12:
+	 *   - MUST set `timestamp` to the number of seconds since Midnight 1
+	 *    January 1970, UTC.
+	 */
+	sent->inv->timestamp = tal(sent->inv, u64);
+	*sent->inv->timestamp = time_now().ts.tv_sec;
+
+	/* If they don't specify an invoice_timeout, make it the same as we're
+	 * prepare to wait. */
+	if (!invoice_timeout)
+		invoice_timeout = timeout;
+	else if (*invoice_timeout < *timeout)
+		return command_fail(cmd, JSONRPC2_INVALID_PARAMS,
+				    "invoice_timeout %u must be >= timeout %u",
```

Comment:
`invoice_timeout %u must be >= timeout %u` -> `invoice_timeout %u must be >= offer timeout %u`